src/domains/sign/99_variables.tf

```diff
@@ -43,11 +43,18 @@ variable "tags" {
 
 # domain specific
 
+variable "subnets" {
```

Comment:
May be is better to call `subnets_cirds`, or create a separated variable for each cird. To be more clear